<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormHome</class>
 <widget class="QWidget" name="FormHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>731</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/images/cidaHome2.jpeg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLineEdit">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>267</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="acccederPushButton">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>360</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Acceder</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passLineEdi">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>300</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelContrasena">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="olvidoPasswordpushButton_2">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>330</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 10px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
    color:rgb(0, 0, 116);
    border-style: inset;
}

QPushButton:pressed {
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>Crear Cuenta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="olvidoPasswordpushButton">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>307</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
     border-style: inset;
     border-width: 0px;
     border-radius: 0px;
     border-color: beige;
	 border-color: rgb(217, 217, 217);
     font: bold 10px;
     min-width: 10em;
     text-align:left;
 }

QPushButton:hover{
    color:rgb(0, 0, 116);
    border-style: inset;
}

QPushButton:pressed {
     border-style: inset;
 }

</string>
   </property>
   <property name="text">
    <string>¿Olvido su contraseña?</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
